crates/gosub_styling/tools/generate_definitions/main.go

```diff
@@ -41,7 +42,30 @@ func main() {
 		len(webrefData.Properties), len(webrefData.Values), len(webrefData.AtRules), len(webrefData.Selectors),
 	)
 
+	sort.Slice(webrefData.Properties, func(i, j int) bool {
+		return webrefData.Properties[i].Name < webrefData.Properties[j].Name
+	})
+
 	for _, property := range webrefData.Properties {
+
+		if property.Syntax == "" {
+			alias, err := webref.GetAlias(property.Name)
+			if err != nil {
+				log.Panic("failed to get alias syntax for property: " + property.Name)
+			}
+
+			for wdp := range webrefData.Properties {
+				if webrefData.Properties[wdp].Name == alias {
+					property.Syntax = webrefData.Properties[wdp].Syntax
```

Comment:
I wouldn't straight up copy the syntax, I think it makes sense to also add a property that it is an alias and which prop it aliases. So we can copy the value in the matching part directly to the aliased prop.

So when we have `-webkit-mask`, we copy the property to `mask` in the matcher. Basically at the same point where we resolve shorthands.

This makes it easier in the mask implementation, since we only need to check for `mask-clip` and not also `-webkit-mask-clip`

Reply:
that might make sense

Reply:
i'll move the aliases to a separate "alias" listing, and pick them up in the matcher